Dockerfile

```diff
@@ -35,22 +35,17 @@ FROM debian:${DEBIAN_TAG} as debian-base
 
 # Install packages needed for running Atlantis.
 # We place this last as it will bust less docker layer caches when packages update
-# hadolint ignore explanation
-# DL3008 (pin versions using "=") - Ignored to avoid failing the build
-# SC2261 (multiple redirections) - This is a bug https://github.com/hadolint/hadolint/issues/782
-# hadolint ignore=DL3008,SC2261
```

Comment:
you need this comments back otherwise hadolint will complain

Reply:
oops! I thought the comment meant the opposite, that you shouldn't pin the images 🤦